src/components/canvas/Ball.jsx

```
@@ -7,30 +7,40 @@ import {
+      rotationIntensity={0.75} // Reduced rotation intensity
+      floatIntensity={1.5} // More subtle floating effect
+      floatingRange={[0.05, 0.15]} // Defined float range
+    >
+      <ambientLight intensity={0.5} />
+      <directionalLight 
+        position={[1, 2, 3]} 
+        intensity={1.5}
+        castShadow
+      />
+      <mesh castShadow receiveShadow scale={2.5}>
+        <icosahedronGeometry args={[1, 4]} /> // Increased geometry detail
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)